addons/website/static/src/builder/plugins/options/navtabs_images_style_option.xml

```diff
@@ -5,9 +5,23 @@
             <attribute name="classAction">'s_tabs_nav_vertical'</attribute>
             <attribute name="id">'vertical_opt'</attribute>
         </xpath>
+        <xpath expr="//BuilderRow[@label.translate=&quot;Style&quot;]" position="after">
+            <BuilderContext t-if="isActiveItem('pills_opt')" applyTo="'.s_tabs_pills_template .nav-pills'">
+                <BuilderRow label.translate="Background" level="1">
+                    <BuilderColorPicker enabledTabs="['solid', 'custom', 'gradient']" styleAction="'--nav-pills-link-active-bg'"/>
+                </BuilderRow>
+                <BuilderRow label.translate="Text" level="1">
+                    <BuilderColorPicker enabledTabs="['solid', 'custom']" styleAction="'--nav-pills-link-active-color'"/>
+                </BuilderRow>
+            </BuilderContext>
+        </xpath>
         <xpath expr="//BuilderRow[@label.translate=&quot;Direction&quot;]" position="after">
             <BuilderRow label.translate="Descriptions" t-if="isActiveItem('vertical_opt')">
-                <BuilderCheckbox classAction="'s_tabs_nav_with_descriptions'"/>
+                <BuilderSelect>
+                    <BuilderSelectItem classAction="'s_tabs_nav_with_descriptions'">All items</BuilderSelectItem>
```

Comment:
@Brieuc-brd would be good to create a task so that the description are not saved in the DOM if they are hidden :grimacing: It's apparently like that since the start (cc @SergeBayet) :pray: Doing something like we do for the pricelist snippet: during edit mode, toggling description on/off only hides them in the DOM, but saving the DOM actually removes/adds them (assuming this still works today..?).

So necessary important to do for this to move forward though.